NaviGator/utils/voltage_gui/src/voltage_gui.py

```diff
@@ -33,8 +52,16 @@ def __init__(self, context):
         context.add_widget(self.myWidget)
         self.runGUI()
 
-    # Updates Values displayed in GUI every second
     def runGUI(self) -> None:
+        """
+        Updates Values displayed in GUI every second
+        
+        Args:
+            No arguments are passed in. 
+        
+        Returns: 
+            There is no present return statement.
```

Comment:
Hi Pranet, thanks for making these changes. For functions that return `None`, or return nothing, we usually don't document a "Returns:" field for these functions. We can just omit this section of the function doc.

Reply:
This applies for all other functions that return `None` in this file and others